packages/web/src/js/TextAreaGrow.ts

```diff
@@ -0,0 +1,69 @@
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+import SelectorEngine from './dom/SelectorEngine';
+import EventHandler from './dom/EventHandler';
+
+const NAME = 'textAreaGrow';
```

Comment:
```suggestion
const NAME = 'textArea';
```

Reply:
The concept is to have the same name plugin as is the component and add the features to them. E. g. the `grow` is a feature of the `TextArea` plugin.